Sources/Swift/Integrations/SessionReplay/SentryOnDemandReplay.swift

```diff
@@ -44,20 +45,46 @@ class SentryOnDemandReplay: NSObject, SentryReplayVideoMaker {
     
     var videoWidth = 200
     var videoHeight = 434
+    var videoScale: Float = 1
     var bitRate = 20_000
     var frameRate = 1
     var cacheMaxSize = UInt.max
     
+    private var actualWidth: Int { Int(Float(videoWidth) * videoScale) }
+    private var actualHeight: Int { Int(Float(videoHeight) * videoScale) }
+        
+    init(outputPath: String, workingQueue: SentryDispatchQueueWrapper, dateProvider: SentryCurrentDateProvider) {
+        self._outputPath = outputPath
+        self.dateProvider = dateProvider
+        self.workingQueue = workingQueue
+    }
+        
+    convenience init(withContentFrom outputPath: String, workingQueue: SentryDispatchQueueWrapper, dateProvider: SentryCurrentDateProvider) {
+        self.init(outputPath: outputPath, workingQueue: workingQueue, dateProvider: dateProvider)
+        guard let content = try? FileManager.default.contentsOfDirectory(atPath: outputPath) else { return }
```

Comment:
Replace FileManager.default.contentsOfDirectory with something we already have in SentryFileManager

Reply:
There is nothing in SentryFileManager to retrieve content of a directory

Reply:
See `allFilesInFolder`, which calls `contentsOfDirectoryAtPath` and does some basic error handling.

Reply:
I see. This function was not in the header. 
Anyway, I prefer not to add `SentryFileManager` dependency to `SentryOnDemanReplay`.